lib/tesla_api/auth/login.ex

```diff
@@ -184,17 +184,16 @@ defmodule TeslaApi.Auth.Login do
           String.contains?(body, "Recaptcha is required") ->
             {:error, %Error{reason: :recaptcha_required, env: env}}
 
-          String.contains?(body, "/oauth2/v3/authorize/mfa/verify") ->
+          String.contains?(body, "/authorize/mfa/verify") ->
```

Comment:
I have to change this as telsa.cn uses /oauth2/v1/authorize/mfa/verify in javascript